### DISCUSSION TITLE
Request => Street Fighter: The Storytelling Game

### DISCUSSION THREAD

**Discussion by rdsbrazil** [JohanFalt/Foundry_WoD20#486] [Ideas]
Could you give some love for this underrated classic? 
Just a  character sheet would be enough!

**Comment by JohanFalt** [comment]
I could yes, but not very quickly. I have quite alot planned in the coming versions (see Issues). So please create an issue with it and I will plan it in.

**Comment by JohanFalt** [comment]
Created an issue based on this discussion.

**Comment by JohanFalt** [comment]
https://github.com/JohanFalt/Foundry_WoD20/issues/492
